### DISCUSSION TITLE
Update the example folder

### DISCUSSION THREAD

**Discussion by AmjadGhzlangit** [chillerlan/php-qrcode#269] [Ideas]
Hi,

First, thank you for the awesome package! It's been very helpful.

However, I noticed that the example in svgWithLogoAndCustomShapes.php in the examples folder still uses the outputInterface field, which is now deprecated. This caused some confusion and I spent more than an hour trying to figure out why it wasn't working.

Could you please update the examples to reflect the current best practices and avoid using deprecated fields? It would be incredibly helpful for users like myself who are trying to implement these features.

Thanks again for your hard work and support!

Best regards,
Amjad

![image_2024-05-29_16-06-24](https://github.com/chillerlan/php-qrcode/assets/96890984/6b30806f-b855-4326-bf2b-7c181d6f0102)

**Comment by codemasher** [comment]
Hi, the example is fine, the `$outputInterface` field is marked deprecated because as of v6 it will not be nullable anymore, see #223:

https://github.com/chillerlan/php-qrcode/blob/0a897ca7235757a6e59e0949157daa5e8f7c680b/src/QROptionsTrait.php#L128-L136


That's the current dev-main:

https://github.com/chillerlan/php-qrcode/blob/0ca3d2377f8d01b788c738d46f9bf72d9b17083d/src/QROptionsTrait.php#L100-L103
